src/core/gui/XournalppCursor.cpp

```diff
@@ -515,6 +520,44 @@ auto XournalppCursor::createHighlighterOrPenCursor(int size, double alpha) -> Gd
 }
 
 
+auto XournalppCursor::createHorizontalLineCursor(int size, double alpha, GdkWindow* theWindow) -> GdkCursor* {
+    gint x, y;
+    GdkDevice *mouse_device;
+
+    #if GTK_CHECK_VERSION (3,20,0)
+    GdkSeat *seat = gdk_display_get_default_seat (gdk_display_get_default ());
+    mouse_device = gdk_seat_get_pointer (seat);
+    #else
+    GdkDeviceManager *devman = gdk_display_get_device_manager (gdk_display_get_default ());
+    mouse_device = gdk_device_manager_get_client_pointer (devman);
+    #endif
+
+    gdk_window_get_device_position (theWindow, mouse_device, &x, &y, NULL);
+
+    int height = size;
+    int width = gdk_window_get_width(theWindow);
+
+    int centerY = height / 2;
+    cairo_surface_t* crCursor = cairo_image_surface_create(CAIRO_FORMAT_ARGB32, width, height);
+    cairo_t* cr = cairo_create(crCursor);
+
+    cairo_set_line_width(cr, 1.2);
+    cairo_move_to(cr, 0, centerY);
+    cairo_line_to(cr, width, centerY);
```

Comment:
Why not just use y-coordinate 0? Then you don't have to translate it later on when creating the cursor from the pixbuf.

Reply:
I don't think I understand. I thought that by using centerY, the cursor would appear centered over where your mouse actually is. I suppose I don't see any reason _not_ to use 0, though.